lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -138,6 +138,7 @@ def event_action_params(event)
       if @action == 'update'
         params[:_upsert] = LogStash::Json.load(event.sprintf(@upsert)) if @upsert != ""
         params[:_script] = event.sprintf(@script) if @script != ""
+        params[:_retry_on_conflict] = @retry_on_conflict
```

Comment:
This is send to the elasticsearch ruby client right? I though this options was without an underscore?

https://github.com/elastic/elasticsearch-ruby/blob/master/elasticsearch-api/lib/elasticsearch/api/actions/update.rb#L50 

Maybe I don't point to the right doc? I am a bit confused.